paper-typeahead-input.html

```diff
@@ -473,8 +504,8 @@
       }
     },
     _search:function(term){
-      if (term == ""){
-        this._suggestions = [];
+      if (term == "" && !this.isAllowSuggestOnEmptyInput){
```

Comment:
This case for suggest items when no chars in the input. When you dont know
the first letter of your item - you can look on existing.
Our case is order number. You must choose from top 10 IDs.

On 18 July 2016 at 18:06, Tianxiang Zhang <notifications@github.com> wrote:

> In paper-typeahead-input.html
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30#discussion_r71164813>
> :
>
> > @@ -473,8 +504,8 @@
> >        }
> >      },
> >      _search:function(term){
> > -      if (term == ""){
> > -        this._suggestions = [];
> > +      if (term == "" && !this.isAllowSuggestOnEmptyInput){
>
> Can you give a use case to search on empty string?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30/files/dfdc34b0472906d1f9c7005d4442b0162ab8c4f8#r71164813>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG0SnJefU7h_e_NhbmavzVUB1mAnyz2Gks5qW5Z4gaJpZM4JKWUL>
> .
>



-- 
C уважением, Пличко Павел